addons/website/static/src/scss/website.scss

```diff
@@ -909,7 +909,7 @@ $-transition-duration: 200ms;
             }
             @include media-breakpoint-up(lg) {
                 .navbar-brand {
-                    font-size: 0;
+                    @include font-size(0);
```

Comment:
Not to be used here: first it probably does not work properly to use the mixin inside a media query environment. But for forcing a font-size to 0, it really should not be used.